py3status/modules/backlight.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+"""
+Display the current screen backlight level.
+
+Configuration parameters:
+    cache_timeout: how often we refresh this module in seconds (default: 10s)
+    color:  The text color (default: "#FFFFFF")
+    device: The backlight device
+            If not specified the plugin will try to detect it automatically
+
+Format status string parameters:
+    {level} brightness
+
+@author Tjaart van der Walt (github:tjaartvdwalt)
+@license BSD
+"""
+
+from __future__ import division
+
+import os
+from time import time
+
+
+def get_device_path():
+    for (path, devices, files) in os.walk('/sys/class/backlight/'):
+        for device in devices:
+            if 'brightness' in os.listdir(path + device) and \
+               'max_brightness' in os.listdir(path + device):
+                return path + device
+
+
+class Py3status:
+    """
+    """
+    # available configuration parameters
+    cache_timeout = 10
+    color = '#FFFFFF'
+    format = u'☼: {level}%'
+
+    def backlight(self,  i3s_output_list, i3s_config):
+        if(not hasattr(self, 'device')):
+            device_path = get_device_path()
+        else:
+            device_path = "/sys/class/backlight/%s" % self.device
```

Comment:
Can we not just remember the device_path nrather than having to calculate it every time?